src/libs/antares/sys/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+set(PROJ sys)
+set(HEADERS
+        include/antares/${PROJ}/appdata.h
+        include/antares/${PROJ}/policy.h
+        include/antares/${PROJ}/hostname.hxx
+)
+set(SRC_PROJ
+        ${HEADERS}
+        policy.cpp
+        appdata.cpp
+)
+source_group("memory" FILES ${SRC_PROJ})
+
+add_library(${PROJ} ${SRC_PROJ})
+add_library(Antares::${PROJ} ALIAS ${PROJ})
+
+target_link_libraries(${PROJ}
+        PRIVATE
+        yuni-static-core
```

Comment:
yuni must be PUBLIC because the public headers of sys include some headers of yuni